pkg/kube/inject/inject.go

```diff
@@ -572,6 +573,13 @@ func InjectionData(sidecarTemplate, valuesConfig, version string, typeMetadata *
 		"toLower":             strings.ToLower,
 	}
 
+	// Allows the template to use env variables from istiod.
+	// Istiod will use a custom template, without 'values.yaml', and the pod will have
+	// an optional 'vendor' configmap where additional settings can be defined.
+	funcMap["env"] = func(key string, def string) string {
```

Comment:
default is not used?

Also nit, this can probably be inline in the funcMap declaration?

Reply:
It is used now.

There is nothing else in the funcMap declaration that inlines stuff - consistency.